python/upload_templates.py

```diff
@@ -0,0 +1,207 @@
+    recording_asset = dandiset.get_asset_by_path(path=asset_path)
+    url = recording_asset.get_content_url(follow_redirects=True, strip_query=True)
+    file_path = url
+
+    electrical_series_path = "acquisition/ElectricalSeriesAp00"
+    recording = NwbRecordingExtractor(
+        file_path=file_path,
+        stream_mode="remfile",
+        electrical_series_path=electrical_series_path,
+    )
+    session_id = recording._file["general"]["session_id"][()].decode()
+    eid = session_id.split("-chunking")[0]  # eid : experiment id
+
+    ONE.setup(base_url="https://openalyx.internationalbrainlab.org", silent=True)
+    one_instance = ONE(password="international")
+
+    pids, probes = one_instance.eid2pid(eid)
+
+    # Let's select the probe
+
+    probe_number = electrical_series_path.split("Ap")[-1]
+
+    sorting_pid = None
+    for pid, probe in zip(pids, probes):
+        probe_number_in_pid = probe[-2:]
+        if probe_number_in_pid == probe_number:
+            sorting_pid = pid
+            break
+
+    sorting = IblSortingExtractor(
+        pid=sorting_pid, one=one_instance, good_clusters_only=True
+    )
+
+
+
+    ibl_recording = IblRecordingExtractor(pid=pid, stream_name=f"probe{probe_number}.ap" )  
+    probe_info = ibl_recording.get_annotation("probes_info")
+    
+    folder_path = "./nwb_recording/"
+    recording_cache = recording.save_to_folder(
+        folder=folder_path, overwrite=True, n_jobs=-1, chunk_memory="5G"
+    )
+
+    pre_processed_recording = common_reference(
+        highpass_filter(
+            phase_shift(astype(recording=recording_cache, dtype="float32")),
+            freq_min=1.0,
+        )
+    )
+
+    # take first and last minute
+    sampling_frequency_recording = pre_processed_recording.sampling_frequency
+    sorting_sampling_frequency = sorting.sampling_frequency
+    num_samples = pre_processed_recording.get_num_samples()
+
+    # Take the last 60 minutes of the recording
+    minutes = 60
+    samples_before_end = int(minutes * 60.0 * sampling_frequency_recording)
+
+    start_frame_recording = num_samples - samples_before_end
+    end_frame_recording = num_samples
+
+    recording_end = pre_processed_recording.frame_slice(
+        start_frame=start_frame_recording, end_frame=end_frame_recording
+    )
+
+    # num_samples = sorting.get_num_frames()
+    samples_before_end = int(minutes * 60.0 * sorting_sampling_frequency)
+    start_frame_sorting = num_samples - samples_before_end
+    end_frame_sorting = num_samples
+
+    sorting_end = sorting.frame_slice(
+        start_frame=start_frame_sorting, end_frame=end_frame_sorting
+    )
+
+    analyzer = create_sorting_analyzer(
+        sorting_end, recording_end, sparse=False, folder=f"analyzer_{eid}"
+    )
+
+    random_spike_parameters = {
+        "method": "all",
+    }
+
+    template_extension_parameters = {
+        "ms_before": 3.0,
+        "ms_after": 5.0,
+        "operators": ["average", "std"],
```

Comment:
```suggestion
        "operators": ["average"],
```
I guess we don't need the std here, right?

Reply:
right

Reply:
OK.